### PR TITLE
Update Jest tests, replace enzyme with react-testing-library

### PR DESCRIPTION
**Pull request by gsreynolds**
- Update previously skipped tests
- Replace enzyme with react-testing-library
- Update tests for react-testing-library

### CODE REVIEW

**Comment by gsreynolds** [comment]
Makes #231 redundant by removing that dependency

File: src/mocks/store.test.js [hunk 1]
```diff
@@ -1,30 +1,6 @@
...
-
-export const waitForComponentToPaint = async (wrapper) => {
-  await act(async () => {
-    await new Promise((resolve) => setTimeout(resolve, 10));
-    wrapper.update();
-  });
-};
+export default mockStore;
 
 // Ref: https://stackoverflow.com/a/59864054/6480733 - required for test mocking
```
Comment (gsreynolds) [inline src/mocks/store.test.js:6]:
```suggestion
```

**Review by gsreynolds — COMMENTED** [review]
Remove this line

**Review by martindstone — APPROVED** [review]
Looks good

**Pull request merged**